tools/analysis_tools/get_flops.py

```diff
@@ -43,46 +50,129 @@ def parse_args():
     return args
 
 
-def main():
-    args = parse_args()
+def inference(args, logger):
+    if str(torch.__version__) < '1.12':
+        logger.warning('In torch<1.12, torch.jit don\'t supprt'
+                       'some configs compute flops like configs/detectors'
+                       ',configs/yolact, if you want to count the'
+                       'flops mentioned above, you need update torch version. '
+                       'other configs can compute flops in torch<1.12')
```

Comment:
Some config files, such as configs/yolact, may have compatibility issues with torch.jit when torch<1.12. If you want to calculate flops for these models, please make sure your pytorch version is >=1.12.